src/js/main.js

```diff
@@ -0,0 +1,42 @@
+// self invoking function
+(function () {
```

Comment:
Je mag alles in dezelfde self invoking function steken.

Daarbinnen kan je zo veel functies aanmaken zoals je wil, maar dan delen de variabelen dezelfde [scope](https://www.w3schools.com/js/js_scope.asp)

Reply:
Done!